src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -317,6 +317,7 @@ public synchronized void applyDDL(Sequenced<DDLEvent> sequencedEvent) throws Exc
 
     latestOffset = event.getOffset();
     latestSequenceNum = sequenceNumber;
+    LOG.debug("DDL offset: {} seq num: {}", latestOffset.get(), latestSequenceNum);
```

Comment:
nit: remove this.

Reply:
as discussed offline, change it to trace level.